### PR TITLE
Be review

### PR DESCRIPTION
**Pull request by nastasi-oq**
Enable review process with email notification to dataset owner of changes performed by reviewer.

Tests are green: https://ci.openquake.org/job/zdevel_open-risk-data-dashboard/28

---
User "reviewer_user" reviewed your dataset.

It passed the review phase.
### Dataset with changes:

Field Name | Current | Previous
-- | -- | --
country | Mexico | (no changes)
url | http://emidius.mi.ingv.it/CPTI15-DBMI15/    <br>     https://www.riskopendata.eu/conrtibute.html |(no changes)
tag | Tsunami <br>  Coastal flooding <br>  Landslide <br>  Earthquake | Tsunami <br> Coastal flooding <br> Landslide <br> Earthquake | (no Changes)
is reviewed | True | (no changes)
notes | hhhh | (no changes)
is existing | False | (no changes)
is existing txt | et voilà | (no changes)
is digital form | True | (no changes)
Is available Online | False | (no changes)
is avail online meta | True | (no changes)
is bulk avail | False | (no changes)
is machine read | True | (no changes)
is machine read txt | zorrozzc | zorrozzcsdsd
is pub available | True | (no changes)
is avail for free | True | (no changes)
is open licence | True | (no changes)
is open licence txt |   | (no changes)
is prov timely | False | (no changes)
is prov timely last |   | (no changes)
keydataset | HA_10: Coastal Flooding and tsunamis - Nearshore tsunami wave heigths - Tsunami wave heights nearshore - International | (no changes)

### CODE REVIEW

File: backend/ordd_api/__init__.py [hunk 1]
```diff
@@ -1 +1,2 @@
-__VERSION__="0.11.1"
+__VERSION__="0.12.0"
+MAIL_SUBJECT_PREFIX="R.O.D.I."
```
Comment (daniviga) [inline backend/ordd_api/__init__.py]: Why this is here and not in config?

File: backend/ordd_api/models.py [hunk 2]
```diff
@@ -255,7 +255,7 @@ class Dataset(models.Model):
     is_existing = models.BooleanField()
     is_existing_txt = models.TextField(blank=True, null=False)
     is_digital_form = models.BooleanField()
-    is_avail_online = models.BooleanField()
+    is_avail_online = models.BooleanField("Is available Online")
```
Comment (daniviga) [inline backend/ordd_api/models.py]: Why this and only on that field?

File: backend/ordd_api/views.py [hunk 3]
```diff
@@ -173,6 +169,7 @@ def get_queryset(self):
             owner=self.request.user)
 
     def perform_create(self, serializer):
+        print("perform_create: BEGIN")
```
Comment (daniviga) [inline backend/ordd_api/views.py]: Please add a fixme for debug prints

File: backend/ordd_api/models.py (hunk 2 above)
Comment (nastasi-oq) [inline backend/ordd_api/models.py]: It was a test (and it works) a complete list of description will be provided in a different PR

File: backend/ordd_api/__init__.py (hunk 1 above)
Comment (nastasi-oq) [inline backend/ordd_api/__init__.py]: Not a specific reason.

File: backend/ordd_api/views.py (hunk 3 above)
Comment (nastasi-oq) [inline backend/ordd_api/views.py]: Removed at all.
